### ISSUE TITLE
region rule should ignore content of noscript elements

### ISSUE THREAD

**Issue by nico3333fr** [dequelabs/axe-core#864]
Hi there, 

I've used it on https://van11y.net/ thanks to Sonarwhal https://sonarwhal.com/scanner/39c2fa0b-cfaa-49c9-bc5f-3564a2eaec21 

It reported me this error: "Content should be contained in a landmark region"

It seems (look at the thread on Twitter https://twitter.com/molant/status/989497275134001154) that it is due to:

```
<noscript><p class="invisible"><img src="/piwik/piwik.php?idsite=1" alt="" /></p></noscript>
```

This piece of code is for Matomo Analytics (ex-Piwik). It seems to be a false positive.

I hope this will help you to improve your tool. 

Cheers,
Nicolas

**Comment by jkodu** [comment]
@WilcoFiers 
I was looking to pick this up and the first point of fix was to introduces a `region-matches.js` for region rule and ignore `noscript` and its children. 

But your comment reads, `axe-core` should ignore `noscript`. Am I right in thinking we need to make this a global filter of some sort, even before executing the matches? 

Thoughts?

**Comment by jkodu** [comment]
Not able to reproduce the issue.
